ledger/tracker.go

```diff
@@ -98,19 +116,73 @@ type ledgerForTracker interface {
 	GenesisHash() crypto.Digest
 	GenesisProto() config.ConsensusParams
 	GenesisAccounts() map[basics.Address]basics.AccountData
+
+	// TODO: temporary?
+	scheduleCommit(basics.Round)
+	waitAccountsWriting()
 }
 
 type trackerRegistry struct {
 	trackers []ledgerTracker
+	driver   *accountUpdates
+
+	// ctx is the context for the committing go-routine. It's also used as the "parent" of the catchpoint generation operation.
+	ctx context.Context
```

Comment:
I don't see that this context object is the parent of the catchpoint generation operation; did I miss it ?

Reply:
I found out I do not need it. The `cancelFunc` is called only on `close()`, that calls tracker's `close()` that cancels catchpoint generation as well. I updated the comment